src/OpenTelemetry.Extensions.Enrichment/EnrichmentActions.cs

```diff
@@ -0,0 +1,42 @@
+// <copyright file="EnrichmentActions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+
+namespace OpenTelemetry.Extensions.Enrichment;
+
+#pragma warning disable CA1812 // Class is instantiated through dependency injection
+internal sealed class EnrichmentActions : TraceEnricher
+#pragma warning restore CA1812 // Class is instantiated through dependency injection
+{
+    public static readonly List<Action<TraceEnrichmentBag>> Actions = new();
```

Comment:
This feels a it odd to mix the static here.... Could we add the Actions<TEbag> itself to the DI, and have this ctor() retrieve a list of Actions and apply it one by one?

Reply:
Updated, thanks